src/Extensions/TimedScopes/TimedScope.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.TimedScopes
+{
+	/// <summary>Logs duration of activity</summary>
+	public class TimedScope : IDisposable
+	{
+		/// <summary>TimedScope result</summary>
+		public TimedScopeResult Result { get; set; }
+
+
+		/// <summary>TimedScope sub tipe</summary>
+		public string SubType { get; set; }
+
+
+		/// <summary>TimedScope meta data</summary>
+		public string MetaData { get; set; }
+
+		/// <summary>Indicates if activty was </summary>
```

Comment:
>was [](start = 36, length = 3)

Incomplete sentence. I image it should say "finished" at the end

Reply:
Fixed, thanks